specs/src/SUMMARY.md

```diff
@@ -7,6 +7,7 @@
   - [Namespace](./specs/namespace.md)
   - [Shares](./specs/shares.md)
   - [Consensus](./specs/consensus.md)
+    - [CAT Pool](./specs/cat_pool.md)
```

Comment:
I think we may want to add this to the README here: https://github.com/celestiaorg/celestia-app/blob/492c3db844119da836295ca951d4548dec08133b/specs/src/README.md?plain=1#L7

It's a bit ridiculous to maintain 3 locations for the same table of contents so we may consider deleting that README alltogether.

Reply:
Yeah I added it to the `SUMMARY.md` which I think is the main one in terms of the rendered static site